build.assets/Dockerfile-centos7-fips

```diff
@@ -77,17 +74,19 @@ RUN (curl -L https://github.com/coreos/etcd/releases/download/v3.3.9/etcd-v3.3.9
 # 2) Get source for the correct Go boringcrypto runtime and compile it with Go bootstrap runtime
 # 3) Erase Go bootstrap runtime and create build directories
 # 4) Print compiled Go version
+ARG GO_BOOTSTRAP_RUNTIME=go1.9.7
```

Comment:
@codingllama Do we even need that block anyway? Looks like we were building `boringcrypto` only because of glibc dependency, but because of Rust we require glibc 2.17 anyways.

Reply:
Dropped on https://github.com/gravitational/teleport/pull/16479/commits/e0d47cee133e3b5a646814873e616124304baf57, thanks for pointing it out.